packages/babel-plugin-transform-react-jsx/test/fixtures/react/should-properly-handle-comments-between-props/expected.js

```diff
@@ -0,0 +1,8 @@
+var x = React.createElement("div", {
+  /* a multi-line
+     comment */
+  attr1: "foo"
+}, React.createElement("span", {
+  // a double-slash comment
```

Comment:
This comment changed lines (it used to be printed right after the `{`), but I don't think this breaks the test unless this is supposed to preserve its position.